bpf/lib/nat.h

```diff
@@ -477,6 +478,65 @@ static __always_inline int snat_v4_icmp_rewrite_embedded(struct __ctx_buff *ctx,
 	return 0;
 }
 
+static __always_inline int
+snat_v4_icmp_rewrite_egress_embedded(struct __ctx_buff *ctx,
+				     struct ipv4_ct_tuple *tuple,
+				     struct ipv4_nat_entry *state,
+				     __u32 l4_off, __u32 inner_l4_off)
+{
+	int ret, flags = BPF_F_PSEUDO_HDR;
+	struct csum_offset csum = {};
+	__be32 sum_l4 = 0, sum;
```

Comment:
This syntax is taxing, does `int ret, flags = BPF_F_PSEUDO_HDR;` assign `BPF_F_PSEUDO_HDR` to both `ret` and `flags` or only `flags` and does it leave `ret` uninitialized? Same goes for `sum_l4` and `sum`. Please keep to one variable declaration per line to keep this easy to read.